apps/daq_application.cxx

```diff
@@ -78,6 +78,7 @@ main(int argc, char* argv[])
 
   // Set/Update the application name in the environment. Used by logging/ers.
```

Comment:
Might be a good idea to update this comment

Reply:
done. thanks!